apps/api/src/domain/models/shared/common-commands/grant-resource-read-access-to-user/resource-read-access-granted-to-user.event-handler.ts

```diff
@@ -18,7 +24,7 @@ export class ResourceReadAccessGrantedToUserEventHandler implements ICoscradEven
             },
         } = event;
 
-        if (resourceType !== AggregateType.term) {
+        if (resourceType !== AggregateType.term && resourceType !== AggregateType.photograph) {
```

Comment:
What's going on here? I can't recall doing this for `term`. Let's see why this is here.

Reply:
This isn't good. We need to fix this before merging.

Reply:
I've fixed this now.